DiscordBot.DataAccess/UnsafeEventsSheetsService.cs

```diff
@@ -0,0 +1,539 @@
+﻿using System;
```

Comment:
This has just been moved from EventsSheetsService right, no other changes? 🙂

Reply:
Pretty much, The class name has changed, and some `virtual` keywords were added to make the inheritance work, but those have been removed again in 81c8e0e 🙂